cloudshell/firewall/paloalto/panos/autoload/panos_snmp_system_info.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import re
+
+from cloudshell.snmp.autoload.snmp_system_info import SnmpSystemInfo
+
+
+class PanOSSNMPSystemInfo(SnmpSystemInfo):
+    DEVICE_MODEL_PATTERN = re.compile(r"::pan(?P<model>\S+$)")
+
+    def _get_device_os_version(self):
+        """Get device OS Version form snmp SNMPv2 mib.
+
+        :return: device model
+        :rtype: str
+        """
+        try:
+            result = self._snmp_handler.get_property(
+                "PAN-COMMON-MIB", "panSysSwVersion", "0"
+            ).safe_value
+        except Exception:
```

Comment:
there is no need for try: except clause here. The safe_value method would not return you any error as far as I remember.